bin/devil-bot-rust-cdk.ts

```diff
@@ -7,3 +7,7 @@ const app = new cdk.App();
 new DevilBotRustCdkStack(app, 'DevilBotRustCdkStack', {
     env: { region: app.node.tryGetContext('region') || 'us-east-1' },
 });
+// Dev stage
+new DevilBotRustCdkStack(app, 'DevilBotRustCdkStackDev', {
```

Comment:
You may want to update the `README.md` to deploy to the dev stack only for testing. Right now we advise everyone to run `cdk deploy` which deploys all stacks, but now we probably want to do `cdk deploy DevilBotRustCdkStackDev`.

I also have some other ideas for making this partial CD - I'm going to set up a Code Deploy pipeline and see if there are any safe constructs to disallow users to deploy directly to prod (instead it will have to go through the pipeline) so people can't do so by mistake. I'll make a GitHub issue for this.